sdk/Javascript/src/Microsoft.WindowsAzure.Mobile.JS.csproj

```diff
@@ -1,13 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
   <PropertyGroup>
     <ToolsPath>$(MSBuildProjectDirectory)\..\tools</ToolsPath>
     <MakePriPath Condition="Exists('$(MSBuildProgramFiles32)\Windows Kits\8.0\bin\x64\makepri.exe')">$(MSBuildProgramFiles32)\Windows Kits\8.0\bin\x64\makepri.exe</MakePriPath>
     <MakePriPath Condition="Exists('..\..\Tools\makepri.exe')">..\..\Tools\makepri.exe</MakePriPath>
+    <TargetPlatformVersion>8.1</TargetPlatformVersion>
```

Comment:
What does this drive, I'm not sure I want to say no to 8.0 for Source, only for testing.